VideoLocker/res/values/strings.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">edX</string>
```

Comment:
Note that this string needs to be defined to be referencable, even if you're customizing it from `build.gradle`.